### ISSUE TITLE
Add support for other file formats

### ISSUE THREAD

**Issue by dibyadas** [dibyadas/imagery#8]

**Comment by dibyadas** [comment]
@arunikayadav42 , yeah sure! Just to be clear about what I mean in this issue, I want to make any file that is shared to the Slack workspace shareable via a link. Imagine I post a text file, Imagery can take that and upload it somewhere private and post the link back. So let's start with text file. And since this is a new feature, this will be a bit of a long commitment. Have you setup Imagery in your test slack workspace?

**Comment by arunikayadav42** [comment]
No I haven't done it yet...I will set it up by tonight and then get back?

**Comment by dibyadas** [comment]
Sure. Here is the [wiki](https://github.com/dibyadas/imagery/wiki/Deploying-Imagery-On-Heroku) page that has the detailed instructions for setting it up.

**Comment by arunikayadav42** [comment]
thanks @dibyadas

**Comment by vibhavagarwal5** [comment]
@arunikayadav42 Did u got the environment up and running? 😄

**Comment by arunikayadav42** [comment]
no sir I wasn't able to

**Comment by vibhavagarwal5** [comment]
Do u need some help regarding it? Maybe check the [wiki](https://github.com/dibyadas/imagery/wiki) fo the steps?

**Comment by arunikayadav42** [comment]
Hey,

I just signed the petition "CM Devendra Fadnavis, JICA Stop the cutting of
3500 trees in Aarey Forest" and wanted to see if you could help by adding
your name.

Our goal is to reach 500,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/G92k5LHcYf

Thanks!
Arunika
